src/styles/index.css

```diff
@@ -1 +1,4 @@
-/* new data here */
+body {
+  padding: 4%;
```

Comment:
- [ ] To stick to the template design as said in the requirements I think you don't need this padding in your body. this will make your project have an **EXACTLY**  Styling as the one provided on [Zeplin](https://zeplin.io/)Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)